s3stream/src/main/java/com/automq/stream/s3/wal/impl/block/SlidingWindowService.java

```diff
@@ -62,6 +64,8 @@ public class SlidingWindowService {
      * The lock of {@link #pendingBlocks}, {@link #writingBlocks}, {@link #currentBlock}.
      */
     private final Lock blockLock = new ReentrantLock();
+
+    private final Lock pollBlocKLock = new ReentrantLock();
```

Comment:
The comments for `blockLock` should be updated, and meanwhile, add some comments for `pollBlockLock`.

Reply:
Now `pendingBlocks` is not protected by `blockLock`, comments of `blockLock` should be updated.